benchmark/AlcorPerformanceTestReport/README.md

```diff
@@ -0,0 +1,718 @@
+
+## Alcor K8s cluster set up:  
+
+| Machine                    | CPU                                       | Number of CPU | total vCPU | Memory (GB) |
+| -------------------------- | ----------------------------------------- | ------------- | ---------- | ----------- |
+| Control node 10.213.43.251 | Intel(R) Xeon(R) CPU E7-4809 v3 @ 2.00GHz | 2             | 32         | 32          |
+| Worker node 10.213.43.69   | Intel(R) Xeon(R) CPU E5-2630 v3 @ 2.40GHz | 2             | 32         | 96          |
+| Worker node 10.213.43.70   | Intel(R) Xeon(R) CPU E5-2630 v3 @ 2.40GHz | 2             | 32         | 96          |
+| Worker node 10.213.43.71   | Intel(R) Xeon(R) CPU E5-2630 v3 @ 2.40GHz | 2             | 32         | 96          |
+| Worker node 10.213.43.72   | Intel(R) Xeon(R) CPU E5-2630 v3 @ 2.40GHz | 2             | 32         | 96          |
+| Worker node 10.213.43.73   | Intel(R) Xeon(R) CPU E5-2630 v3 @ 2.40GHz | 2             | 32         | 128         |
+| Worker node 10.213.43.74   | Intel(R) Xeon(R) CPU E5-2630 v3 @ 2.40GHz | 2             | 32         | 128         |
+
+## Alcor Medina (OpenStack) cluster setup:  
+
+Our Alcor Medina (OpenStack) cluster is configured with 5 controller node, and 64 worker (compute) nodes.  
+Below are the list of OpenStack services running on our 5 controller node, all below service have one copy on each machine:  
+- cinder_api_container  
+- galera_container  
+- glance_container  
+- heat_api_container  
+- horizon_container  
+- keystone_container  
+- memcached_container  
+- neutron_agents_container (Not used)  
+- neutron_server_container (Not used)  
+- nova_api_container  
+- placement_container  
+- rabbit_mq_container  
+- repo_container  
+- utility_container  
+
+**Our controller nodes configurations:**  
+| Machine                     | CPU                                       | Number of CPU | total vCPU | Memory (GB) |
+| --------------------------- | ----------------------------------------- | ------------- | ---------- | ----------- |
+| Controller 1: 10.213.43.169 | Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz  | 2             | 32         | 128         |
+| Controller 2: 10.213.43.170 | Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz  | 2             | 32         | 128         |
+| Controller 3: 10.213.43.171 | Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz  | 2             | 32         | 128         |
+| Controller 4: 10.213.43.91  | Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz | 1             | 24         | 32          |
+| Controller 5: 10.213.43.94  | Intel(R) Xeon(R) CPU E5-2620 v3 @ 2.40GHz | 2             | 24         | 128         |
+
+**Our compute nodes configurations:**  
+- In total we have 2,808 vCPUs, 16,263 GB of RAM, and 64,254 GB of Storage
+
+| VCPUs (total) | RAM (total) | Local Storage (total) |
+| ------------- | ----------- | --------------------- |
+| 80            | 755.5GB     | 730GB                 |
```

Comment:
@yanmo96 Please give it one more column before "VCPUs" to indicate the Number or name for computer node, such as "compute1", "compute2", etc.